node-feature-discovery-job.json.template

```diff
@@ -17,6 +17,14 @@
         }
       },
       "spec": {
+        "volumes": [
```

Comment:
@ConnorDoyle Agreed, but how do you do that via k8s?
We do not want to impact how the networking is setup for the pod, we just want to discover what node networking capabilities exist.
Seems to me that going forward, as we wish to discover more capabilities of the host, we are going to run into this type of concern again and again.
If the pod is completely sandboxed then it will be very limited in what it can discover.
To support it being completely open (i.e. privileged) then that opens other doors...maybe this is ok?
Compromise is required, but what is acceptable?
The kernel api via libnl might(?) be an alternative in this case, but we cannot always depend on such a solution being available.